encodings/fastlanes/src/rle/mod.rs

```diff
@@ -115,25 +112,26 @@ impl RLEArray {
     ///
     /// * `values` - Unique values from all chunks
     /// * `indices` - Chunk-local indices from all chunks
-    /// * `value_chunk_offsets` - Start indices for each value chunk.
+    /// * `values_idx_offsets` - Start indices for each value chunk.
     /// * `validity` - Array validity
     /// * `length` - Array length
     pub fn try_new(
         values: ArrayRef,
         indices: ArrayRef,
-        value_chunk_offsets: ArrayRef,
-        validity: Validity,
+        values_idx_offsets: ArrayRef,
         length: usize,
```

Comment:
we need to validate that length is shorter than indices length and is less than 1024 away